aptos-move/aptos-e2e-comparison-testing/src/data_collection.rs

```diff
@@ -76,12 +77,13 @@ impl DataCollection {
             skip_publish_txns,
             dump_write_set,
             skip_source_code,
+            target_account,
         ))
     }
 
     fn execute_transactions_at_version_with_state_view(
         txns: Vec<Transaction>,
-        debugger_stateview: &DebuggerStateView,
+        debugger_stateview: &DataStateView,
```

Comment:
nit: debugger_state_view? also in other places I suppose

Reply:
done